src/components/crud/complete.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports, import/prefer-default-export */
+import { tasks } from '../..';
+import { storeToLocal } from '../storage';
+import { renderAll } from '../render';
+import { clearAll } from '../variables';
+
+const complete = () => {
+  clearAll.addEventListener('click', () => {
+    const unCompletedTasks = tasks.filter((task) => task.completed === false);
+    unCompletedTasks.forEach((task, index) => {
+      task.index = index + 1;
+    });
+
+    storeToLocal(unCompletedTasks);
+    renderAll();
+    window.location.reload();
+  });
+};
```

Comment:
Good job so far your code is working very well.  What do you think if you wrap this piece of code out of the `complete arrow function` and git rid of this function? when you invoke `complete arrow function`  `clearAll.addEventListener` will not be run because it is listening to click event. It would be better if you implement [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).